docs/using.md

```diff
@@ -2,7 +2,7 @@
 
 In this topic you will learn how to use Quix Streams to perform two types of data processing:
```

Comment:
That's a good point. We use topic-based writing, which is why that term is used, but it could be confused with Kafka topics. I will change to something like "In this documentation..." or "In the following sections..."